blog/2021-12-15-new-marketplace/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Creating a Self Updating Marketplace to Showcase all Felvin Instant Apps
```

Comment:
Let's make the title catchy, suggestions:

- "How many instant apps do we have at Felvin? There is an app for that!"
- "An App to show all the apps"
- "How did I create an app to show all the available apps?"